### PR TITLE
Sort the output of crawl by score

### PR DESCRIPTION
**Pull request by alanorwick**

### CODE REVIEW

File: .ci/crawl.sh [hunk 1]
```diff
@@ -20,6 +20,17 @@ do
     echo pids
 done
 
+for net in "${arr[@]}"
+do
+    set -- $net
+    port=$1
+    node_name=$2
+    
+    jq 'to_entries | sort_by(-.value.score) | from_entries' all-nodes/all-$node_name.json > tmp_file.tmp && mv tmp_file.tmp all-nodes/all-$node_name.json
```
Comment (gameofpointers) [inline .ci/crawl.sh:29]: does this do descending order ?

Comment (alanorwick) [reply .ci/crawl.sh:29]: It should be in descending order. Score 50 is better than score 1

Comment (alanorwick) [reply .ci/crawl.sh:29]: Or atleast thats what I thought

Comment (gameofpointers) [reply .ci/crawl.sh:29]: i just meant if `sort_by` does descending order by default

Comment (gameofpointers) [reply .ci/crawl.sh:29]: If it doesn you can merge this

Comment (alanorwick) [reply .ci/crawl.sh:29]: no, it does ascending by default

**Review by gameofpointers — APPROVED** [review]

File: .ci/crawl.sh (hunk 1 above)
Comment (gameofpointers) [reply .ci/crawl.sh:29]: dont we want descending ?

**Pull request merged**